### PR TITLE
Some UI fixes

### PR DESCRIPTION
**Pull request by romefeller**
- Fixed the sig error
- Changed the get public key order
- Still needs error handling

### CODE REVIEW

File: docs/transfer-ledger-create.html [hunk 1]
```diff
@@ -293,39 +304,47 @@
...
      
+     function dimBody(msg){
+        $("body").dimmer({closable:false}).dimmer("show");
+        $(".dimmer").html("<div class='content'> <h3> " + msg + "</h3></div>");
+     }
+     
      async function getLedger(){
         if (!window.ledger) {
-          var transp = await window.TranspWeb.create();
+          var transp = await window.HttpTransp.open("http://localhost:5000/apdu");
```
Comment (ApolloUnicorn) [inline docs/transfer-ledger-create.html:314]: This appears to point to Speculos

File: Dockerfile [hunk 2]
```diff
@@ -4,5 +4,5 @@ ENV GIT_URL https://github.com/obsidiansystems/kadena-transfer-js.git
 
 RUN apt-get -y update \
 && apt-get -y install git \
-&& git clone -b hw-ledger-signing ${GIT_URL} \
+&& git clone -b ago-ui-fix ${GIT_URL} \
```
Comment (ApolloUnicorn) [inline Dockerfile:7]: Should this actually point to `master` or some other fixed branch?

Comment (romefeller) [reply Dockerfile:7]: Yes, master. I was using this to save time.

File: docs/transfer-ledger-create.html (hunk 1 above)
Comment (romefeller) [reply docs/transfer-ledger-create.html:314]: I do not have a ledger. So I had to do this.

**Comment by ApolloUnicorn** [comment]
Closing in favor of #4

**Pull request closed without merging**